AUTHORS.md

```diff
@@ -50,6 +50,7 @@ name is available.
     Bastiaan Olij (BastiaanOlij)
     Ben Brookshire (sheepandshepherd)
     Benjamin Larsson (Nallebeorn)
+    Benjamin Lösch (MJacred)
```

Comment:
```suggestion
```
See the instructions above, you do not edit this manually

Reply:
Ok, will remove. I've seen it done this way before

Reply:
As far as I can find it's never been done by a user before, only a fix to an incorrect name

Reply:
Indeed, only name fixes. Seems I misremembered. Thanks for clearing that up!

Reply:
Done